### ISSUE TITLE
Negative Money?

### ISSUE THREAD

**Issue by Dustyragon** [TurningWheel/Barony#433]
This one is kinda odd. Playing multiplayer with some friend and one dropped gold. However it turns out it was negative gold and.. it actually somehow exists but puts your gold into negatives and drains whatever gold you had when you pick it up.
https://imgur.com/a/iXlqSWu

**Comment by WALLOFJUSTICE** [comment]
Is that reproducible at all?

**Comment by Dustyragon** [comment]
I'm honestly not even sure how it happened in the first place. Friend was trying to level trading so we went to drop him gold and other friend somehow had negative money he dropped. Got no idea how it happened or if it'll happen again.

**Comment by NoahRuppel** [comment]
I know this is an old post, but I've found (with the help of cheats) that any time you manage to get negative money (easier on multiplayer because of lag), you can drop that money, and it will always drop all of the negative money, resetting your counter to zero.

**Comment by Dustyragon** [comment]
Is it a lag based thing? We had a solid connection so I don't really know what happened.

**Comment by NoahRuppel** [comment]
I don't believe so, I used an assembly script to mess with my gold amount, but playing with my little brother we were able to dupe items by throwing them, picking them up in midair, and repeat for about 10 seconds, and I believe the clients just kind of give up for a second, and something similar may have happened on accident in the case of the negative gold. Sadly, unless you have money to spare and can get a super nice computer, there's nothing you can do in the way of that.
